package.json

```diff
@@ -40,7 +40,7 @@
     "test:node:debug": "mocha debug node-tests"
   },
   "dependencies": {
-    "@babel/core": "^7.12.0",
+    "@babel/core": "~7.12.0",
     "@babel/helper-compilation-targets": "^7.12.0",
```

Comment:
@stevetyler  I think you might need to update all @babel  related package versions to 7.12.13 or 7.12.x versions